### PR TITLE
Set prefix for Ansible groups

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

**Comment by dLobatog** [comment]
@agx mind to take a look?

**Comment by agx** [comment]
What about making the prefix foreman_ instead of foreman ? So we can have no prefix as well?

**Comment by dLobatog** [comment]
Sounds better, will change it in a moment :smile:

**Comment by dLobatog** [comment]
@agx Updated!

File: foreman_ansible_inventory.py [hunk 1]
```diff
@@ -112,6 +112,11 @@ def read_settings(self):
 
         self.group_patterns = eval(group_patterns)
 
+        try:
+            self.group_prefix = config.get('ansible', 'group_prefix')
+        except (ConfigParser.NoOptionError, ConfigParser.NoSectionError):
+            self.group_prefix = "foreman"
```
Comment (agx) [inline foreman_ansible_inventory.py]: We need to default to foreman_ here too then. Looks good otherwise!

**Comment by dLobatog** [comment]
@agx Updated the default to foreman_

**Comment by agx** [comment]
Thanks for updating the patch!

**Pull request merged**